src/main/java/io/citrine/jcc/search/pif/result/PifMultiSearchResultElement.java

```diff
@@ -0,0 +1,23 @@
+package io.citrine.jcc.search.pif.result;
+
+import io.citrine.jcc.search.core.result.BaseMultiSearchResultElement;
+
+/**
+ * Class to store the information in a single search result as a part of a PIF multi-query.
+ *
+ * @author Kyle Michel
+ */
+public class PifMultiSearchResultElement extends BaseMultiSearchResultElement<PifSearchResult> {
```

Comment:
The user already has to provide type information: PifMultiSearchResult.  It seems to me like `PifMultiSearchResult` is literally just an alias for `BaseMultiSearchResultElement<PifSearchResult>`.  That's a lot of code for an alias.

As a separate point, it would be pretty great if the user could just say `searchResult = Query.execute()` without specifying the type (since it is defined already by the query).  If only there was a jvm language that could do that...